contrib/libglibutil/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "libglibutil"
+pkgver = "1.0.79"
+pkgrel = 0
+build_style = "makefile"
+make_use_env = True
+hostmakedepends = ["pkgconf"]
+makedepends = ["glib-devel"]
+pkgdesc = "Library of glib utilities"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "BSD-3-Clause"
+url = "https://github.com/sailfishos/libglibutil"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "4d689cb79f8ea061e46b89008370dc771b07164ee496850d9d56d9d85f1be1c3"
+# check: no tests
+options = ["!check"]
+
+
+def install(self):
+    self.do(
+        "make", "STRIP=true", f"DESTDIR={self.chroot_destdir}", "install-dev"
+    )
```

Comment:
```suggestion
def post_install(self):
```

make_install_target = "install-dev"

Reply:
thx!